packages/browser-extension/source/localization/localizationInterfaces.ts

```diff
@@ -0,0 +1,10 @@
+export interface LocaleWithPosition {
```

Comment:
Strange interface name. Maybe `Message` or `LocalizedMessage` would be suitable


interface LocalizedMessage {
   id
   value
}

interface SourcePosition { // this interface is more general we might import it in here
   start
   end
}

type alias XY = LocalizedMessage | SourcePosition